mmcv/transform/loading.py

```diff
@@ -0,0 +1,241 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import mmcv
+import numpy as np
+from .builder import TRANSFORMS
+from .base import BaseTransform
+
+
+@TRANSFORMS.register_module()
+class LoadImageFromFile(BaseTransform):
+    """Load an image from file.
+
+    Required key is "img_path". Added or updated keys are "img", "width",
```

Comment:
Suggest to use sphinx list in docstring, like
```
Required Key:

- img_path

Modified Key:

- img
- width
- height
- ori_width
- ori_height
```

We can check the effectiveness by preview the API doc.